Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -133,10 +135,12 @@ VerifierOptions verifierOptions
       Dafny.Program program,
       bool loadCanceled
     ) {
+      var parseAndResolutionDiagnostics = errorReporter.GetDiagnostics(textDocument.Uri);
       return new DafnyDocument(
         Options,
         textDocument,
-        errorReporter.GetDiagnostics(textDocument.Uri),
+        parseAndResolutionDiagnostics,
+        parseAndResolutionDiagnostics.Any(d => d.Severity == DiagnosticSeverity.Error),
```

Comment:
should this not be `d.Severity != DiagnosticSeverity.Error`? I.e. doesn't this say that verification can be done if there is at least one diagnostic error?

Reply:
What exactly does this method do? Is it called after resolution already happened, so diagnostic errors here will only come from plugins?

Reply:
I just checked, and this method creates a document with no program. It is called in case a plugin fails to load, parsing fails, or loading a document is cancelled by a new version of that document.
So first, you were write, it should have been `.All(d.Severity != DiagnosticSeverity.Error)`, but more generally, since verification is not supposed to run after creating this document, I replaced the condition with `false`